packages/mobile/src/import/ImportWallet.tsx

```diff
@@ -72,7 +72,7 @@ export class ImportWallet extends React.Component<Props, State> {
 
   checkCleanBackupPhrase = () => {
     const { route, navigation } = this.props
-    if (route && route.params.clean) {
+    if (route.params?.clean) {
```

Comment:
This is the bug that triggered the crash when navigating to the import screen.

Reply:
```suggestion
    if (route?.params?.clean) {
```